src/LanguageServer/Impl/LanguageServer.Lifetime.cs

```diff
@@ -24,55 +24,60 @@
 
 namespace Microsoft.Python.LanguageServer.Implementation {
     public partial class LanguageServer {
+        private static readonly TimeSpan s_cancelAfter = TimeSpan.FromMilliseconds(200);
+
         private InitializeParams _initParams;
         private bool _shutdown;
 
         private Task<IDisposable> _initializedPriorityTask;
 
         [JsonRpcMethod("initialize")]
         public async Task<InitializeResult> Initialize(JToken token, CancellationToken cancellationToken) {
-            _initParams = token.ToObject<InitializeParams>();
+            _initParams = ToObject<InitializeParams>(token);
```

Comment:
this one was only one that doesn't use ToObject method to deserialize token.

Reply:
Yeah, its a good catch.